lib/query-types.d.ts

```diff
@@ -1,16 +1,17 @@
+declare enum QueryTypes {
+  SELECT,
+  INSERT,
+  UPDATE,
+  BULKUPDATE,
+  BULKDELETE,
+  DELETE,
+  UPSERT,
+  VERSION,
+  SHOWTABLES,
+  SHOWINDEXES,
+  DESCRIBE,
+  RAW,
+  FOREIGNKEYS,
+}
```

Comment:
this is wrong. Those are not numeric enums, they are strings. A string enum would work

Reply:
@felixfbecker  Thanks for the correction. I'm pushing up a fix now to make this a string enum.

I appreciate the feedback 👍